functionality/database/src/main/kotlin/pl/patrykgoworowski/mintlift/functionality/database/measurement/MeasurementWithLatestEntryView.kt

```diff
@@ -0,0 +1,19 @@
+@file:Suppress("Indentation")
+package pl.patrykgoworowski.mintlift.functionality.database.measurement
```

Comment:
```suggestion
package com.patrykandpatryk.liftapp.functionality.database.measurement
```

I’d also suggest renaming the packages accordingly.